README.md

```diff
@@ -219,6 +222,24 @@ libController.on(libController.ItemMouseLeaveEventName, function(contextData) {
 })
 ```
 
+## Jest test structure 
+
+```typescript
+  // Importing the object to be tested
+import { LibraryItem } from '../src/components/LibraryItem';
+
+// A test case that tests object properties against their intended value 
+it("should create a libraryItem", function () {
+          let libContainer = libController.createLibraryContainer(layoutSpecsJson, loadedTypesJson);
+
+          let libraryItem = mount(<LibraryItem libraryContainer={libContainer} data={data} />); 
+          expect(libraryItem.props().data.childItems[0].text).to.equal("Child0"); 
+
+}
+```
+- Instructions to write a tests are found at https://github.com/DynamoDS/librarie.js/wiki/Setting-up-test-frameworks-for-librarie.js
+
+
```

Comment:
Thanks @monikaprabhu, understood. However, this [still does not look okay](https://github.com/monikaprabhu/librarie.js/blob/338f5d031b417ca51be047d1d180f2efab1822c3/README.md) 😢 

Can you help ensuring:

1. Documentation formatting is proper
2. Move it under `Run Tests` section above

Thanks